versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -63,13 +64,16 @@ class CaffeineCacheBackend implements CacheBackend {
 
     Caffeine<CacheKeyValue, CacheKeyValue> cacheBuilder =
         Caffeine.newBuilder()
+            .scheduler(Scheduler.systemScheduler())
```

Comment:
What benefit are we expecting from this change?

Reply:
I'd like to formally test the effects of this change :)

Reply:
Test included